### DISCUSSION TITLE
Stackblitz example needs updating

### DISCUSSION THREAD

**Discussion by kambanwait** [posva/pinia-colada#107] [General]
The [Stackblitz](https://stackblitz.com/github/posva/pinia-colada-example) example doesn't actually have any functioning code for Pinia Colada in it. I see the files for the store but nothing specific around P.Colada?

Also, I had to run `npm install` and `npm run dev` to get it working. The `pnpm ...` commands don't work and need to be updated

**Comment by posva** [comment]
It does! It's really out of date and doesn't have any demo. I think it's worth migrating the demo of contacts there. It shouldn't be too hard to contribute if anyone is interested: https://github.com/posva/pinia-colada-example

**Reply by kambanwait** [reply]
I can try have a look. Where is this "demo of contacts"? I don't see it in the docs, but I might be missing something here.

Just to confirm, you want the "pinia-colada-example" updated with a pull request, right?

**Reply by posva** [reply]
The demo is in this repo and yes
